setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
+
+setup(
+    name="simpleocr",
+    packages=["simpleocr"],
+    version="1.0.0",
+    description="A library for simple OCR in Python using OpenCV",
+    author="The simple-ocr-opencv authors",
+    url="https://www.github.com/goncalopp/simple-ocr-opencv",
+    download_url="https://www.github.com/goncalopp/simple-ocr-opencv/archive/1.0.0.tar.gz",
```

Comment:
We could make the link point to [goncalopp/simpleocr/releases](https://github.com/goncalopp/simple-ocr-opencv/releases), too. I think that would work just as well.